jmx-metrics/docs/target-systems/solr.md

```diff
@@ -0,0 +1,84 @@
+# Solr Metrics
+
+The JMX Metric Gatherer provides built in Solr metric gathering capabilities.
+
+## Metrics
+
+### Core Metrics
+
+* Name: `solr.document.count`
+* Description: The total number of indexed documents.
+* Unit: `{documents}`
+* Labels: `core`
+* Instrument Type: ObservableLongUpDownCounter
+
+
+* Name: `solr.index.size`
+* Description: The total index size.
+* Unit: `by`
+* Labels: `core`
+* Instrument Type: ObservableLongUpDownCounter
+
+
+* Name: `solr.request.count`
+* Description: The number of queries made.
+* Unit: `{queries}`
+* Labels: `core`, `type`, `handler`
+* Instrument Type: ObservableLongCounter
+
+
+* Name: `solr.request.time.average`
+* Description: The average time of a query. This average is based on the histogram configuration
```

Comment:
nit - this seems like a fairly long description, but has useful information. Maybe something like `The average time of a query, based on Solr's histogram configuration.`? I'm not sure if "EDR" is an accepted abbreviation in dropwizard but I see a few references when searching around.

Reply:
@rmfitzpatrick I have made the change you suggested. I like your shortening.